mods/ra/mod.yaml

```diff
@@ -67,8 +67,6 @@ Chrome:
 Assemblies:
 	./mods/common/OpenRA.Mods.Common.dll
 	./mods/ra/OpenRA.Mods.RA.dll
-	./mods/cnc/OpenRA.Mods.Cnc.dll
```

Comment:
My preference would be to keep the DLL for now, and then replace the overly specific Ion Cannon and Nuke support powers with a more general "Fire an armament" power in mods.common.